projects/igniteui-angular/src/lib/core/styles/components/chips/_chips-theme.scss

```diff
@@ -0,0 +1,245 @@
+////
+/// @group themes
+/// @access public
+/// @author <a href="https://github.com/desig9stein" target="_blank">Marin Popov</a>
+////
+
+/// Chips Theme
+/// @param {Map} $palette [$default-palette] - The palette used as basis for styling the component.
+/// @param {Color} $chip-background [null] - The chips background color.
+/// @param {Color} $chip-hover-background [null] - The chips hover background color.
+/// @param {Color} $text-color [null] - The chips text color.
+/// @param {Color} $text-hover-color [null] - The chips text hover color.
+/// @param {Color} $order-icon-color [null] - The chips order icon color.
+/// @param {Color} $order-icon-hover-color [null] - The chips order icon hover color.
+/// @param {Color} $dir-icon-color [null] - The chips direction icon color.
+/// @param {Color} $remove-icon-color [null] - The chips remove icon color.
+/// @param {Color} $remove-icon-hover-color [null] - The chips remove icon hover color.
+///
+/// @requires extend
+/// @requires text-contrast
+/// @requires igx-color
+///
+/// @example scss Change the background color
+///   $my-chips-theme: igx-chips-theme($chip-background: black);
+///   // Pass the theme to the igx-chips component mixin
+///   @include igx-chips($my-chips-theme);
+@function igx-chips-theme(
```

Comment:
The name should be in the singular form - `igx-chip`, not the plural `igx-chips`. It should be in line with the component selector.